pipeline/stage.go

```diff
@@ -20,10 +22,11 @@ type (
 	//
 	// swagger:model PipelineStage
 	Stage struct {
-		Done  chan error     `json:"-"               yaml:"-"`
-		Name  string         `json:"name,omitempty"  yaml:"name,omitempty"`
-		Needs []string       `json:"needs,omitempty" yaml:"needs,omitempty"`
-		Steps ContainerSlice `json:"steps,omitempty" yaml:"steps,omitempty"`
+		Done        chan error        `json:"-"               yaml:"-"`
+		Environment map[string]string `json:"environment,omitempty" yaml:"environment,omitempty"`
+		Name        string            `json:"name,omitempty"  yaml:"name,omitempty"`
+		Needs       []string          `json:"needs,omitempty" yaml:"needs,omitempty"`
+		Steps       ContainerSlice    `json:"steps,omitempty" yaml:"steps,omitempty"`
```

Comment:
Can we align the `yaml` struct field tags here?

```suggestion
		Done        chan error        `json:"-"                     yaml:"-"`
		Environment map[string]string `json:"environment,omitempty" yaml:"environment,omitempty"`
		Name        string            `json:"name,omitempty"        yaml:"name,omitempty"`
		Needs       []string          `json:"needs,omitempty"       yaml:"needs,omitempty"`
		Steps       ContainerSlice    `json:"steps,omitempty"       yaml:"steps,omitempty"`
```